### PR TITLE
switch to using external clientset in oc rollout retry

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1613144

Switches to using an external clientset in the rollout command.

cc @soltysh @deads2k @jupierce

### CODE REVIEW

File: pkg/oc/cli/rollout/retry.go [hunk 1]
```diff
@@ -35,11 +35,11 @@ type RetryOptions struct {
 	Builder           func() *resource.Builder
 	Mapper            meta.RESTMapper
 	Encoder           runtime.Encoder
-	Clientset         kclientset.Interface
+	Clientset         kexternalclientset.Interface
```
Comment (mfojtik) [inline pkg/oc/cli/rollout/retry.go]: usually I call this "kubernetes.Interface" and drop the import alias.... the "external" client is the new "default", so I don't see a point to name it "external"... (on the other hand, I feel a strong need to name internal clients "internal" ;-)

**Comment by mfojtik** [comment]
one nit, LGTM otherwise

File: pkg/oc/cli/rollout/retry.go [hunk 2]
```diff
@@ -13,8 +13,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	kexternalclientset "k8s.io/client-go/kubernetes"
```
Comment (soltysh) [inline pkg/oc/cli/rollout/retry.go]: `kubernetes` is the preferred alias

File: pkg/oc/cli/rollout/retry.go (hunk 1 above)
Comment (soltysh) [reply pkg/oc/cli/rollout/retry.go]: Honestly, if possible narrow the client to core, only.

**Pull request merged**